docs/rst/tutorials/tools/monitor_with_backend/monitor_with_backend.rst

```diff
@@ -0,0 +1,203 @@
+.. _tutorials_tools_monitor_with_backend:
+
+ROS 2 Monitor example with statistics backend
+=============================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+Background
+----------
+
+Vulcanexus integrates :ref:`vulcanexus_monitor`, which is a useful tool for monitoring and studying a ROS 2 network as ROS 2 relies on the `DDS specification <https://www.omg.org/spec/DDS/1.4/About-DDS/>`_ to communicate the different nodes.
```

Comment:
I'm missing here an extended description of this tutorial, what is it about, why is it userful, etc.